src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -278,8 +284,8 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
             }
 
             const {
-                onError = () => {},
-                onPaymentSelect = () => {},
+                onError = noop,
```

Comment:
Cool!

Reply:
Rather than set default values every time we de-structure, we could set them once within the `_displayWallet` function:

`private async _displayWallet(methodId: string, onPaymentSelect = noop, onError = noop): Promise<void> `

Or, we could avoid setting default values altogether and use optional execution syntax: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls

```tsx
const runOptionalCallback = (optionalCallback?: () => void) => {
    return optionalCallback?.();
}

runOptionalCallback();

runOptionalCallback(() => console.log('I ran'));
```

Reply:
it makes sense to me, i will take you first option if its ok to you